QUICK_START.md

```diff
@@ -6,25 +6,16 @@ This document provides quick access to several different ways to set up CORTX fo
     1. On a single node: [LINK](doc/ova/2.0.0/PI-4/CORTX_on_Open_Virtual_Appliance_PI-4.rst)
     1. On an AWS EC2 instance: [LINK](doc/integrations/AWS_EC2/README.md)
     1. Across a cluster: TODO
-1. Build entire CORTX into binaries: [LINK](doc/community-build/Building-CORTX-From-Source-for-SingleNode.md)
-    1. Run it on a single node: [LINK](doc/community-build/ProvisionReleaseBuild.md)
-    1. Run it on a AWS: [LINK](doc/community-build/cloud/AWS/README.md)
-    1. Run it on a cluster: TODO
+1. Run it on AWS in Kubernetes: [LINK](https://github.com/Seagate/cortx-k8s/blob/stable/doc/cortx-aws-k8s-installation.md)
+2. Build entire CORTX into binaries: [LINK](./doc/community-build/docker/cortx-all/README.md)
```

Comment:
This shouldn't be a 2. All numbers should be 1 and the github server will render them automatically when they serve the page.